scripts/runClangFormat.sh

```diff
@@ -1,16 +1,12 @@
-#!/usr/bin/env bash
+#!/bin/sh
 #
 # Copyright 2020-2022 The Mumble Developers. All rights reserved.
 # Use of this source code is governed by a BSD-style license
 # that can be found in the LICENSE file at the root of the
 # Mumble source tree or at <https://www.mumble.info/LICENSE>.
 
-currentDir=$(dirname $0)
+set -o pipefail
 
-mainDir="$currentDir/.."
-
-cd "$mainDir"
-
-find -type f \( -iname "*.cpp" -o -iname "*.c" -o -iname "*.hpp" -o -iname "*.h" -o -iname "*.cxx" -o -iname "*.cc" \) \
-	-a -not -ipath "./3rdparty/*" -a -not -path "./.git/*" -a -not -ipath "./build/*" \
-	| xargs clang-format --style=file -i
+git ls-files --cached --modified --others -z -- \
+    :^3rdparty/ '**/*'.{cpp,c,hpp,h,cxx,cc} |
+	xargs -0 -- clang-format --style=file -i
```

Comment:
For this I get the error
```
error: cannot use -i when reading from stdin.
```

Reply:
How/where did you run this?
Must've been a directory without source files.
Either way, `xargs -r` fixes it.

Reply:
Yes I ran it from the repo root (which is what I usually do, so great that it is fixed now :+1:)